conans/client/tools/scm.py

```diff
@@ -218,7 +218,7 @@ class SVN(SCMBase):
 
     def __init__(self, folder=None, runner=None, *args, **kwargs):
         def runner_no_strip(command):
-            return decode_text(subprocess.check_output(command, shell=True))
+            return decode_text(subprocess.check_output(command, shell=True, stderr=PIPE))
```

Comment:
Shouldn't be `stderr=STDOUT`?

Reply:
You are right, if we `strerr=STDOUT` we can get errors when we expect stdout, not a good idea. Probably the origin of #4850